deploy/kik-testnet/datadog-agent/etc/datadog-agent/conf.d/json_check.yaml

```diff
@@ -1,7 +1,172 @@
 init_config:
-    default_timeout: 5
+  default_timeout: 5
 
 instances:
-    - url:
-      metrics:
-        - metric_name: jsonpath
+#    - url:
+#      metrics:
+#        - metric_name: jsonpath
+
+#History archive:
+#  - url: bucketURL/.well-known/stellar-history.json
+#    metrics:
+#      - archive.latest_ledger: currentLedger # Latest ledger updated in the history archive
+
+#Horizon metrics:
+# - url: https://horizon.stellar.org/metrics
+#   metrics:
+#   # Metrics that include "5_min_rate" report the 5 minute moving avg pre second of this metric
+#     - horizon.goroutines: goroutines.value # goroutines used by the horizon app
+#     - horizon.elder_ledger: '"history.elder_ledger".value' # oldest ledger in the horizon db
+#     - horizon.latest_ledger: '"history.latest_ledger".value' # ewest ledger in the horizon db
+#     - horizon.open_connections: '"history.open_connections".value' # open connections to the horizon database
+#     - horizon.logging_error: '"logging.error"."5_min_rate"' # error logs
+#     - horizon.logging_panic: '"logging.panic"."5_min_rate"' # error logs
+#     - horizon.logging_warning: '"logging.warning"."5_min_rate"' # warning logs
+#     - horizon.requests_failed: '"requests.failed"."5_min_rate"' # failed requests to horizon (400<= status code < 600)
+#     - horizon.requests_succeeded: '"requests.succeeded"."5_min_rate"' # successful requests to horizon (200<= status code < 400)
```

Comment:
i think it's actually < 300

Reply:
According to this, success it between 200 and 400
https://github.com/stellar/go/blob/master/services/horizon/internal/middleware_request_metrics.go#L21